MIMWebClient/Core/Events/LoadRoom.cs

```diff
@@ -413,12 +413,9 @@ public static void ReturnRoom(Player player, Room room, string commandOptions =
                                 HubContext.SendToClient("You look into the " + itemDescription.name + " and see:", player.HubGuid);
 
 
-                                foreach (var containerItem in itemDescription.containerItems)
+                                foreach (var containerItem in itemDescription.containerItems.List())

```

Comment:
Pass true in the itemcontainer list so we get the article showing before the item.